### PR TITLE
Implements the DB

### PR DESCRIPTION
**Pull request by FerHinojosa**
Implements the DB.

### CODE REVIEW

File: src/main/java/com/jalasoft/webservice/db/DBConnection.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/db/DBConnection.java:10]: This space line does'nt need to be here.

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/db/QueryManager.java:10]: This space line does'nt need to be here.

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        }
+        return query;
+    }
+
+    /**
+     * This classes gethPath the checksum if exist.
+     * @param checksum
+     * @return a String with the path if exist.
+     */
+
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/db/QueryManager.java:56]: This space line does'nt need to be here.

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    public String getPath (String checksum) {
+
+        Statement state = null;
+        ResultSet resultSet = null;
+        String path = "";
+        try{
+            String sql = "Select path from FileStorage where trim(checksum)='" + checksum +"';";
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/db/QueryManager.java:63]: The words Select and where need to be uppercase or lowercase . Select is starting with uppercase.

**Review by ramalaso — COMMENTED** [review]
Please review.

File: src/main/java/com/jalasoft/webservice/db/DBConnection.java (hunk 1 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/db/DBConnection.java:10]: Done

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java (hunk 2 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/db/QueryManager.java:10]: Done

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java (hunk 3 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/db/QueryManager.java:56]: Done

File: src/main/java/com/jalasoft/webservice/db/QueryManager.java (hunk 4 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/db/QueryManager.java:63]: done

**Review by ramalaso — APPROVED** [review]
Thanks.

**Pull request merged**